R/dependencies.R

```diff
@@ -191,9 +191,8 @@ dependencies <- function(
 renv_dependencies_impl <- function(
   path = getwd(),
   root = NULL,
-  ...,
   field = NULL,
-  progress = TRUE,
+  progress = FALSE,
```

Comment:
Doesn't this mean we no longer print progress reports for regular usages?

Reply:
No, because the default for `dependency()` is still `TRUE`

Reply:
Ohh got it!